src/Vox/Admin/Controller/PostController.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * @author  Marco Troisi <hello@marcotroisi.com>
+ * @created 02.12.14
+ */
+
+namespace Vox\Admin\Controller;
+
+use Moss\Http\Response\Response;
+use Moss\Http\Response\ResponseRedirect;
+use Moss\Kernel\AppInterface;
+
+class PostController
+{
+
+    protected $app;
+    protected $security;
+    protected $flashbag;
+    protected $view;
+    protected $request;
+
+    public function __construct(AppInterface $app)
+    {
+        $this->app = $app;
+        $this->request = $this->app->request();
+        $this->security = $this->app->get('security');
+        $this->flashbag = $this->app->get('flashbag');
+        $this->view = $this->app->get('view');
+    }
+
+    public function indexAction()
+    {
+        echo "test";
+    }
+
+    public function editAction()
+    {
+        $this->view->set('post_id', $this->request->query()->get('id'));
+
+        $this->view->template('Vox:Admin:Post:edit');
+
+        return new Response($this->view->render());
+    }
+
+    /**
+     * Returns list of Posts
+     *
+     * @return Response
+     */
+    public function listAction()
+    {
+        return new Response(
```

Comment:
Maybe create class `\Vox\Response\ResponseJSON` extending `Response` object that accepts array as content so we don't this each time.